src/libraries/System.IO/tests/IndentedTextWriter.cs

```diff
@@ -243,6 +244,14 @@ public override Task FlushAsync()
                 return result;
             }
 
+            public override Task FlushAsync(CancellationToken cancellationToken)
+            {
+                Task result = base.FlushAsync();
+                LastCalledMethod = nameof(FlushAsync) + "Cancelable";
```

Comment:
[Cancellable](https://www.merriam-webster.com/dictionary/cancel)? Definitely not blocking since it's a test, but I was expecting the American version to be used here, with double L.

Reply:
We've standardized on all forms having one `L` except for "cancellation" which has two, so "cancel", "canceled", "canceling", "canceler", "cancelable", "cancellation".

Reply:
Got it. Good to know.